services/motion/builtin/builtin_test.go

```diff
@@ -737,59 +737,44 @@ func TestReplanning(t *testing.T) {
 	motionCfg := &motion.MotionConfiguration{PositionPollingFreqHz: 100, ObstaclePollingFreqHz: 1, PlanDeviationMM: epsilonMM}
 
 	type testCase struct {
-		name           string
-		noise          r3.Vector
-		expectedReplan bool
+		name            string
+		noise           r3.Vector
+		expectedSuccess bool
+		expectedErr     string
 	}
 
+	extra := make(map[string]interface{})
+	extra["replan_cost_factor"] = 10.0 // Sensor drift means this will grow
+	replanCount := 4
+	extra["max_replans"] = replanCount
```

Comment:
should we have tests for the default?

Reply:
It's hard to *fully* test the default, since the default is designed to run indefinitely.

I added a test with the default that triggers a replan and then fails due to the cost factor being too low; which at least tests that when defaults are set, we are in fact calling `Replan()` and evaluating the return.